src/invidious/routes/api/v1/misc.cr

```diff
@@ -42,6 +42,15 @@ module Invidious::Routes::API::V1::Misc
     format = env.params.query["format"]?
     format ||= "json"
 
+    listen = env.params.query["listen"]?
+    listen ||= "false"
+
+    if listen == "true" || listen == "1"
+      listen = true
+    else
+      listen = false
+    end
+
```

Comment:
You can directly save the result of the comparison (= a boolean) to a variable! This is much more compact ^^
```suggestion
    listen = env.params.query["listen"]? || "false"
    listen = (listen == "true" || listen == "1")
```

Reply:
Good point, I changed this slightly to eliminate the `|| "false"` (since it wasn't needed). I also split the listen variable in two, since in retrospect, changing the type of a variable feels worse than adding one extra name to the code.